rr_light/reflection_refraction.py

```diff
@@ -1,7 +1,9 @@
 from distutils.util import strtobool
 import math
 from matplotlib import pyplot as plt
-import typing
+from typing import List, Optional
+
+from matplotlib.patches import FancyArrow
```

Comment:
If you don't use this, please remove it.

Reply:
It's removed